<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dashboard</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>5</y>
      <width>580</width>
      <height>545</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="toolTipDuration">
     <number>-2</number>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tabTimeSht">
     <attribute name="title">
      <string>Time Sheets</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>110</y>
        <width>201</width>
        <height>351</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QPushButton" name="btnWTSSetupC">
         <property name="text">
          <string>Setup Week TS (copy prev.)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnOvtSetupC">
         <property name="text">
          <string>Setup OT Week Tab (copy prev.)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnOvtHrsC">
         <property name="text">
          <string>Time Sheet -&gt; OT/Summary</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnPopSapC">
         <property name="text">
          <string>Populate SAP Hours</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="labelDav_2">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>40</y>
        <width>200</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>24</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>DAV</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="Line" name="line_2">
      <property name="geometry">
       <rect>
        <x>288</x>
        <y>40</y>
        <width>4</width>
        <height>410</height>
       </rect>
      </property>
      <property name="baseSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
      <property name="midLineWidth">
       <number>4</number>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_4">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>110</y>
        <width>201</width>
        <height>351</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QPushButton" name="btnWTSSetupD">
         <property name="text">
          <string>Setup Week TS (copy prev.)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnOvtSetupD">
         <property name="text">
          <string>Setup OT Week Tab (copy prev.)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnOvtHrsD">
         <property name="text">
          <string>Time Sheet -&gt; OT/Summary</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnPopSapD">
         <property name="text">
          <string>Populate SAP Hours</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="labelCov_2">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>40</y>
        <width>200</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>24</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>COV</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QProgressBar" name="progressBar_ts">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>460</y>
        <width>461</width>
        <height>23</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777213</height>
       </size>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabEngHrs">
     <attribute name="title">
      <string>Eng Hours</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>80</y>
        <width>461</width>
        <height>381</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="btnEngHrs">
         <property name="text">
          <string>CN47N Active Projects -&gt; Eng Hours</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnEngShp">
         <property name="text">
          <string>Eng Hours Teco -&gt; As Shipped</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QProgressBar" name="progressBar_eh">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>460</y>
        <width>461</width>
        <height>23</height>
       </rect>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabProjDash">
     <attribute name="title">
      <string>Project Dashboard</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget_5">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>80</y>
        <width>461</width>
        <height>381</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QPushButton" name="btnEng2Prj">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="baseSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string notr="true">Both EngHours -&gt; Project Dashboard</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnEco">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>1</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="baseSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="contextMenuPolicy">
          <enum>Qt::CustomContextMenu</enum>
         </property>
         <property name="text">
          <string>Project Dashboard Update ECO</string>
         </property>
         <property name="iconSize">
          <size>
           <width>14</width>
           <height>20</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnRelData">
         <property name="text">
          <string>Generate: Release Data, Due in 2 Weeks, &amp;&amp; Eng Dashboard</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QProgressBar" name="progressBar">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>460</y>
        <width>461</width>
        <height>23</height>
       </rect>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="alignment">
       <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
      </property>
      <property name="textDirection">
       <enum>QProgressBar::TopToBottom</enum>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabMisc">
     <attribute name="title">
      <string>Misc.</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget_6">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>200</y>
        <width>461</width>
        <height>261</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QPushButton" name="btnWinReg">
         <property name="baseSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="focusPolicy">
          <enum>Qt::WheelFocus</enum>
         </property>
         <property name="text">
          <string notr="true">Fix Winshuttle Register Issue</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnClsXl">
         <property name="text">
          <string>Close all Excel Instances (Use if script fails for unknown reason)</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_7">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>110</y>
        <width>201</width>
        <height>151</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="QPushButton" name="btnRemSpcC">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>1</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="baseSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="contextMenuPolicy">
          <enum>Qt::CustomContextMenu</enum>
         </property>
         <property name="text">
          <string>Remove Time Sheet Spaces</string>
         </property>
         <property name="iconSize">
          <size>
           <width>14</width>
           <height>20</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_8">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>110</y>
        <width>201</width>
        <height>151</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <widget class="QPushButton" name="btnRemSpcD">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>1</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="baseSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="contextMenuPolicy">
          <enum>Qt::CustomContextMenu</enum>
         </property>
         <property name="text">
          <string>Remove Time Sheet Spaces</string>
         </property>
         <property name="iconSize">
          <size>
           <width>14</width>
           <height>20</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="labelDav_3">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>40</y>
        <width>200</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>24</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>DAV</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="labelCov_3">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>40</y>
        <width>200</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>24</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>COV</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="Line" name="line_3">
      <property name="geometry">
       <rect>
        <x>288</x>
        <y>40</y>
        <width>4</width>
        <height>180</height>
       </rect>
      </property>
      <property name="baseSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
      <property name="midLineWidth">
       <number>4</number>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QProgressBar" name="progressBar_m">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>460</y>
        <width>461</width>
        <height>23</height>
       </rect>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnEngShp</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>UpdEngShp()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>112</x>
     <y>414</y>
    </hint>
    <hint type="destinationlabel">
     <x>8</x>
     <y>317</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnOvtHrsC</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>CovOvtHrs()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>89</x>
     <y>388</y>
    </hint>
    <hint type="destinationlabel">
     <x>7</x>
     <y>476</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnOvtHrsD</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>DavOvtHrs()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>510</x>
     <y>388</y>
    </hint>
    <hint type="destinationlabel">
     <x>593</x>
     <y>468</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnOvtSetupC</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>CovOvtSetup()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>81</x>
     <y>310</y>
    </hint>
    <hint type="destinationlabel">
     <x>7</x>
     <y>264</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnOvtSetupD</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>DavOvtSetup()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>458</x>
     <y>310</y>
    </hint>
    <hint type="destinationlabel">
     <x>591</x>
     <y>264</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnEngHrs</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>UpdEngHrs()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>86</x>
     <y>284</y>
    </hint>
    <hint type="destinationlabel">
     <x>5</x>
     <y>216</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnWTSSetupC</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>CovWTSSetup()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>99</x>
     <y>220</y>
    </hint>
    <hint type="destinationlabel">
     <x>6</x>
     <y>156</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnWTSSetupD</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>DavWTSSetup()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>516</x>
     <y>232</y>
    </hint>
    <hint type="destinationlabel">
     <x>593</x>
     <y>157</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnPopSapC</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>CovPopSap()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>93</x>
     <y>466</y>
    </hint>
    <hint type="destinationlabel">
     <x>9</x>
     <y>293</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnPopSapD</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>DavPopSap()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>521</x>
     <y>466</y>
    </hint>
    <hint type="destinationlabel">
     <x>593</x>
     <y>359</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnEng2Prj</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>EngH2ProjD()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>78</x>
     <y>251</y>
    </hint>
    <hint type="destinationlabel">
     <x>6</x>
     <y>118</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnEco</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>UpdEco()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>521</x>
     <y>348</y>
    </hint>
    <hint type="destinationlabel">
     <x>593</x>
     <y>293</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnRelData</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>RelData2Wks()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>79</x>
     <y>445</y>
    </hint>
    <hint type="destinationlabel">
     <x>7</x>
     <y>401</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnRemSpcC</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>CovRemSpc()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>89</x>
     <y>246</y>
    </hint>
    <hint type="destinationlabel">
     <x>7</x>
     <y>189</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnRemSpcD</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>DavRemSpc()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>510</x>
     <y>236</y>
    </hint>
    <hint type="destinationlabel">
     <x>594</x>
     <y>201</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnWinReg</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>WinReg()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>106</x>
     <y>327</y>
    </hint>
    <hint type="destinationlabel">
     <x>7</x>
     <y>346</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnClsXl</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>ClsXl()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>82</x>
     <y>444</y>
    </hint>
    <hint type="destinationlabel">
     <x>5</x>
     <y>455</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>UpdEngHrs()</slot>
  <slot>CovOvtHrs()</slot>
  <slot>DavOvtHrs()</slot>
  <slot>CovOvtSetup()</slot>
  <slot>DavOvtSetup()</slot>
  <slot>UpdEngShp()</slot>
  <slot>CovWTSSetup()</slot>
  <slot>DavWTSSetup()</slot>
  <slot>CovPopSap()</slot>
  <slot>DavPopSap()</slot>
  <slot>EngH2ProjD()</slot>
  <slot>UpdEco()</slot>
  <slot>RelData2Wks()</slot>
  <slot>CovRemSpc()</slot>
  <slot>DavRemSpc()</slot>
  <slot>WinReg()</slot>
  <slot>ClsXl()</slot>
 </slots>
</ui>
